sharepoint/sharepoint-ps/Microsoft.Online.SharePoint.PowerShell/Set-SPOTenant.md

```diff
@@ -5495,6 +5500,29 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -StopAlerts
+This parameter allows turning off classic SharePoint Alerts ahead of the announced deprecation on July 1st 2026.
+
+The valid values are:
+
+* False (default) - The classic SharePoint Alerts feature is not turned off.
+* True - The classic SharePoint Alerts feature is turned off and no classic Alert emails are send.
+
+> [!NOTE]
+> This setting supports the [classic SharePoint Alerts retirement](https://aka.ms/retirement/alerts/support) as it allows administrators to disable classic SharePoint Alerts before the feature retires on July 1st 2026.
+
```

Comment:
this is being redundant as it's repeating a bunch of stuff already stated. I'd remove this NOTE entirely, and instead update line 5504:

```
...on July 1, 2026. See the [SharePoint Alerts retirement](https://aka.ms/retirement/alerts/support) page for more information.
```